addons/mail/static/tests/web/follower_list_menu_tests.js

```diff
@@ -289,6 +289,34 @@ QUnit.test("Load 100 recipients at once", async (assert) => {
     assert.containsNone($, ".o-mail-RecipientList span:contains(Load more)");
 });
 
+QUnit.test("Load recipient without email", async (assert) => {
+    const pyEnv = await startServer();
+    const [partnerId_1, partnerId_2] = pyEnv["res.partner"].create([
+        { name: "Luigi" },
+        { name: "Mario" },
+    ]);
+    pyEnv["mail.followers"].create([
+        {
+            is_active: true,
+            partner_id: pyEnv.currentPartnerId,
+            res_id: partnerId_1,
+            res_model: "res.partner",
+        },
+        {
+            is_active: true,
+            partner_id: partnerId_2,
+            res_id: partnerId_1,
+            res_model: "res.partner",
+        },
+    ]);
+    const { openFormView } = await start();
+    await openFormView("res.partner", partnerId_1);
+    await click("button:contains(Send message)");
+    assert.containsOnce($, "span[title='no email address']:contains('Mario')");
+    assert.containsOnce($, "button[title='Show all recipients']");
+    await click("button[title='Show all recipients']");
```

Comment:
`assert.containsOnce($, "button[title='Show all recipients']");` is unnecessary: following `click` is on same selector so it already behaves as a `contains()` on it.